File: src/main/java/at/fhhagenberg/sqe/api/BuildingFloorService.java
package at.fhhagenberg.sqe.api;

import at.fhhagenberg.sqe.entity.BuildingFloor;

import java.rmi.RemoteException;
import java.util.List;

public interface BuildingFloorService {
    public List<BuildingFloor> getAll(int totalNumberOfFloors) throws RemoteException;
    public BuildingFloor get(int floorNumber) throws RemoteException;
}

File: src/main/kotlin/at/fhhagenberg/sqe/api/BuildingFloorService.kt
package at.fhhagenberg.sqe.api

import at.fhhagenberg.sqe.entity.BuildingFloor
import java.rmi.RemoteException

interface BuildingFloorService {
    @Throws(RemoteException::class)
    fun getAll(): List<BuildingFloor>

    @Throws(RemoteException::class)
    fun get(floorNumber: Int): BuildingFloor?
}